group_boards.go

```diff
@@ -0,0 +1,251 @@
+		return nil, resp, err
+	}
+
+	return gbl, resp, err
+}
+
+// GetGroupIssueBoardList gets a single issue board list.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/group_boards.html#single-board-list
+func (s *GroupIssueBoardsService) GetGroupIssueBoardList(gid interface{}, board, list int, options ...OptionFunc) (*BoardList, *Response, error) {
+	group, err := parseID(gid)
+	if err != nil {
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("groups/%s/boards/%d/lists/%d",
+		url.QueryEscape(group),
+		board,
+		list,
+	)
+
+	req, err := s.client.NewRequest("GET", u, nil, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	gbl := new(BoardList)
+	resp, err := s.client.Do(req, gbl)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return gbl, resp, err
+}
+
+// CreateGroupIssueBoardListOptions represents the available
+// CreateGroupIssueBoardList() options.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/group_boards.html#new-board-list
+type CreateGroupIssueBoardListOptions struct {
+	LabelID *int `url:"label_id" json:"label_id"`
+}
+
+// CreateGroupIssueBoardList creates a new issue board list.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/group_boards.html#new-board-list
+func (s *GroupIssueBoardsService) CreateGroupIssueBoardList(gid interface{}, board int, opt *CreateGroupIssueBoardListOptions, options ...OptionFunc) (*BoardList, *Response, error) {
+	group, err := parseID(gid)
+	if err != nil {
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("groups/%s/boards/%d/lists", url.QueryEscape(group), board)
+
+	req, err := s.client.NewRequest("POST", u, opt, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	gbl := new(BoardList)
+	resp, err := s.client.Do(req, gbl)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return gbl, resp, err
+}
+
+// UpdateGroupIssueBoardListOptions represents the available
+// UpdateGroupIssueBoardList() options.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/group_boards.html#edit-board-list
+type UpdateGroupIssueBoardListOptions struct {
+	Position *int `url:"position" json:"position"`
+}
+
+// UpdateIssueBoardList updates the position of an existing
+// group issue board list.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/group_boards.html#edit-board-list
+func (s *GroupIssueBoardsService) UpdateIssueBoardList(gid interface{}, board, list int, opt *UpdateGroupIssueBoardListOptions, options ...OptionFunc) ([]*BoardList, *Response, error) {
+	group, err := parseID(gid)
+	if err != nil {
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("groups/%s/boards/%d/lists/%d",
+		url.QueryEscape(group),
+		board,
+		list,
+	)
+
+	req, err := s.client.NewRequest("PUT", u, opt, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var gbl []*BoardList
+	resp, err := s.client.Do(req, gbl)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return gbl, resp, err
+}
+
+// DeleteGroupIssueBoardList soft deletes a group issue board list. Only for admins and
```

Comment:
Sorry to be nitting here, but seems you forgot this line... 

As well as some lines that contain that the doc links (earlier in this file). The links are of course always a single line, but then they could be written as:

```
// GitLab API docs:
// https://docs.gitlab.com/ce/api/group_boards.html#delete-a-board-list
```